cmd/devtool/README.md

```diff
@@ -13,9 +13,9 @@ An on-chain workflow testing tool that supports the following:
 docker pull livepeer/geth-with-livepeer-protocol:confluence
 docker run -p 8545:8545 -p 8546:8546 --name geth-with-livepeer-protocol livepeer/geth-with-livepeer-protocol:confluence
 
-# Mac M1 ONLY
-# docker pull darkdragon/geth-with-livepeer-protocol:streamflow
-# docker run -p 8545:8545 -p 8546:8546 --name geth-with-livepeer-protocol darkdragon/geth-with-livepeer-protocol:streamflow
+# To pull a specific architecture (amd64 or arm64):
+# docker pull --platform linux/arm64 livepeer/geth-with-livepeer-protocol:confluence
+# docker pull --platform linux/amd64 livepeer/geth-with-livepeer-protocol:confluence
```

Comment:
```suggestion
```

I think we don't need this part, since Docker automatically pulls the right image.